src/cli.coffee

```diff
@@ -15,18 +9,25 @@ options = docopt """
   """
 
 
-generateConfig = ->
-  break for extension in extensions when options[extension]
+{docopt} = require 'docopt'
+options = docopt usage
+
+
+generateConfig = (extension) ->
+  asyncHandlers = require 'async-handlers'
+  fsExtra = require 'fs-extra'
+  path = require 'path'
   src = path.join __dirname, '..', 'config', "default.#{extension}"
   destFilename = "dependency-lint.#{extension}"
   dest = path.join process.cwd(), destFilename
   callback = (err) ->
     asyncHandlers.exitOnError err
     console.log "Configuration file generated at \"#{destFilename}\""
-  fs.copy src, dest, callback
+  fsExtra.copy src, dest, callback
 
 
 if options['--generate-config']
-  generateConfig()
+  break for extension in extensions when options[extension]
+  generateConfig extension
```

Comment:
This seems an unrelated changes, can you extract it out of this PR, so that it is easier to really only see the changes from inlining requires here?